packages/@cosmos-building-blocks/network/test/tgw-connect.test.ts

```diff
@@ -0,0 +1,56 @@
+import { SynthUtils } from '@aws-cdk/assert';
+import '@aws-cdk/assert/jest';
+import '@aws-cdk/core';
+import { App, Stack } from '@aws-cdk/core';
+import { Vpc, SubnetType } from '@aws-cdk/aws-ec2';
+import { TransitGateway, TransitGatewayAttachment, ResolverRule, ResolverRuleAssociation } from '../src';
+
+const app = new App();
+const testStack = new Stack(app, 'TestStack');
+const myvpc = new Vpc(testStack, 'TestVpc', {
+  cidr: '10.180.7.0/24',
+  maxAzs: 1,
+  subnetConfiguration: [
+    {
+      name: 'Main',
+      subnetType: SubnetType.ISOLATED,
+      cidrMask: 26,
+    },
+    {
+      name: 'Redis',
+      subnetType: SubnetType.ISOLATED,
+      cidrMask: 28,
+    },
+  ],
+});
+const gateway = TransitGateway.fromGatewayAttributes(testStack, 'TransitGateway', {
+  gatewayId: 'tgw-0d4180d3e7c919164',
+});
+const attachment = new TransitGatewayAttachment(testStack, 'TransitGatewayAttachment', {
+  gateway: gateway,
+  vpc: myvpc,
+  subnets: [{ subnetType: SubnetType.ISOLATED }],
```

Comment:
Probably better to test by selecting only Main subnet and see if creates attachment in only one subnet or not

Reply:
done

Reply:
cool